src/fragments/start/getting-started/android/build-footer.mdx

```diff
@@ -4,3 +4,5 @@
 </InternalLinkButton>
 
 [Already have AWS resources? (Cognito, S3, etc.)](/lib/project-setup/use-existing-resources)
+
+Use Amplify Studio to get started with Amplify. [Get started](https://sandbox.amplifyapp.com/getting-started)
```

Comment:
@salihgueler I'm confused as to what these edits in the build-footer.mdx files are. Do you know?

Reply:
Previous pages were not stating the Flutter logo affiliation text properly. It was showing it before a content instead of stating it at the end.